src/javascript/app_2/Modules/SmartChart/Containers/smart_chart.jsx

```diff
@@ -62,23 +63,24 @@ class Chart extends React.Component {
 }
 
 Chart.propTypes = {
-    barriers_array  : PropTypes.array,
-    chart_id        : PropTypes.number,
-    chart_type      : PropTypes.string,
-    end_epoch       : PropTypes.number,
-    granularity     : PropTypes.number,
-    InfoBox         : PropTypes.node,
-    is_mobile       : PropTypes.bool,
-    is_title_enabled: PropTypes.bool,
-    markers_array   : PropTypes.array,
-    onSymbolChange  : PropTypes.func,
-    onUnmount       : PropTypes.func,
-    settings        : PropTypes.object,
-    start_epoch     : PropTypes.number,
-    symbol          : PropTypes.string,
-    wsForget        : PropTypes.func,
-    wsSendRequest   : PropTypes.func,
-    wsSubscribe     : PropTypes.func,
+    barriers_array    : PropTypes.array,
+    chart_id          : PropTypes.number,
+    chart_type        : PropTypes.string,
+    end_epoch         : PropTypes.number,
+    granularity       : PropTypes.number,
+    InfoBox           : PropTypes.node,
+    is_mobile         : PropTypes.bool,
+    is_title_enabled  : PropTypes.bool,
+    markers_array     : PropTypes.array,
+    onSymbolChange    : PropTypes.func,
+    onUnmount         : PropTypes.func,
+    settings          : PropTypes.object,
+    showLastDigitStats: PropTypes.bool,
```

Comment:
why is a boolean not starting with `is_` or `should_`? also why is it in camelCase which is for functions instead of in lowercase and separated by `_`?